recipes/triqs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "2.2.1" %}
+
+package:
+  name: triqs
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/triqs/releases/download/{{ version }}/triqs-{{ version }}.tar.gz
+  sha256: 523d60a98af0c07e972338f3a5f07d7ea1232d6e67b0f063e139d986f09b9864
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - boost-cpp
+    - fftw
+    - gmp
+    - hdf5
+    - h5py
+    - mako
+    - mpi4py
+    - numpy  1.14
+    - openmpi
+    - python
+    - scipy
+    - libblas
+    - liblapack
+    - libclang
+    - zlib
+  run:
+    - boost-cpp
+    - clangxx >=8
```

Comment:
Do you need this at runtime? Or just `libclang` ?

Reply:
I just need libclang but we need a recent compiler for the build. Should I move it up to build instead?

Reply:
> I just need libclang but we need a recent compiler for the build. Should I move it up to build instead?

Let me know how to proceed to guarantee `clangxx >=8` at build time.

Reply:
Yes, add `clangxx >=8  # [osx]` to `build` and remove from here